hera_cal/abscal.py

```diff
@@ -1457,15 +1459,22 @@ def cut_bls(datacontainer, bls, bl_cut):
 
     bls : dictionary, keys are antenna-pair tuples and values are baseline vectors in meters
 
-    bl_cut : float, maximum baseline separation [meters] to keep in data
+    min_bl_cut : float, minimum baseline separation [meters] to keep in data
+
+    max_bl_cut : float, maximum baseline separation [meters] to keep in data    
 
-    Output (cut_datacontainers)
+    Output
     ------
     cut_datacontainer : DataContainer object with bl cut enacted
     """
+    if min_bl_cut is None:
+        min_bl_cut = 0.0
+    if max_bl_cut is None:
+        max_bl_cut = 1e10
     cut_datacontainer = odict()
```

Comment:
Because data containers now carry around some metadata, I think this is a bad solution. Instead, I'd make a deepcopy and use `del`.

Reply:
Also, there may be other places where you turn dicts/odicts into DataContainers. We should avoid that if possible.